src/components/Router/MainRouter.tsx

```diff
@@ -94,7 +94,24 @@ export default class MainRouter extends Component<InternalRouterProps, State> {
       steps && steps.some(isUploadFallbackOffAndShouldUseCamera)
     const { hasCamera } = this.props
 
-    return !isDesktop && !hasCamera && shouldStrictlyUseCamera === true
+    const faceStep = steps.find(
+      (step) => step.type === 'face'
+    ) as StepConfigFace
```

Comment:
Since [this PR](https://github.com/onfido/onfido-sdk-ui/pull/1373), we switched to use `buildStepFinder` and `useSdkOptions` hooks for type-guarded step picker from `steps` array.

```suggestion
    const findStep = buildStepFinder(steps)
    const faceStep = findStep('face')
```

Another example for this popular use case: https://github.com/onfido/onfido-sdk-ui/blob/5b8b5ca6533ad58610a946371cb9afac33f3dc1b/src/components/Router/HistoryRouter.tsx#L225-L226